sage_codes/Point computation 

```diff
@@ -1,4 +1,4 @@
-\\\ base point compututation
+/// base point compututation in Montgomery form
```

Comment:
Comments should appear with `#` in front, not `///` as this is not a Rust file.

Reply:
Check it for the whole file.

Reply:
I shall